README.md

```diff
@@ -7,8 +7,9 @@ Provides a lightweight Swift Logging framework.
 
 ## Features:
 
+- Output logs to the stdout stream by default, or choose another output stream by extending the standard library
```

Comment:
Logs output to stdout by default. You can change the output stream, see example usage for [`HeliumStreamLogger.use(_:outputStream:)`](http://ibm-swift.github.io/HeliumLogger/Classes/HeliumStreamLogger.html#use(_:outputStream)).